transport-native-epoll/src/main/c/netty_epoll_native.c

```diff
@@ -315,7 +315,7 @@ static jstring netty_epoll_native_kernelVersion(JNIEnv* env, jclass clazz) {
 }
 
 static jboolean netty_epoll_native_isSupportingSendmmsg(JNIEnv* env, jclass clazz) {
-    if (sendmmsg) {
+    if (&sendmmsg) {
```

Comment:
But if the function is present, the compiler will think the if is always true and warn.  The function signature is only weak if that preprocessor define is set.  In my case, it is not defined as weak, which makes the function always non NULL.

The code proposed has the same problem.  If you try compiling this library with -Werror,-Wpointer-bool-conversion on a recent Linux kernel I think you will see what I am talking about.